### PR TITLE
IDEM-SITE-Lighthouse

### PR DESCRIPTION
**Pull request by brettvniekerk**
This is PR is for adding Lighthouse to the CI process. Using it to test out how it functions as part of a FE spike.

### CODE REVIEW

**Review by xiangzhe0812 — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -36,3 +37,14 @@ jobs:
           node-version: "14"
       - run: yarn
       - run: yarn test
+  lighthouse:
+    runs-on: ubuntu-18.04
```
Comment (bitcoinbrisbane) [inline .github/workflows/ci.yml:41]: Should we aim for at least ubuntu 20?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -36,3 +37,14 @@ jobs:
...
+    needs: build
+    name: Lighthouse
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
```
Comment (bitcoinbrisbane) [inline .github/workflows/ci.yml:48]: And make these 16?

**Review by bitcoinbrisbane — COMMENTED** [review]
Few suggestions, otherwise LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (brettvniekerk) [reply .github/workflows/ci.yml]: changed to use `ubuntu-latest` 👍

File: .github/workflows/ci.yml (hunk 2 above)
Comment (brettvniekerk) [reply .github/workflows/ci.yml]: 👍

**Review by xiangzhe0812 — DISMISSED** [review]

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -5,35 +5,35 @@ on:
...
   lint:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     name: Lint
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: "14"
+          node-version: "16"
```
Comment (xcjlo985) [inline .github/workflows/test.yml:14]: Grab the node version like [this](https://github.com/dltxio/nestjs-template/blob/main/.github/workflows/build-project.yml)

**Review by xcjlo985 — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 3 above)
Comment (brettvniekerk) [reply .github/workflows/test.yml]: 👍

**Review by bitcoinbrisbane — APPROVED** [review]

**Review by xcjlo985 — APPROVED** [review]
Looking forward to a showcase/walkthrough and socialising to the mesh

**Pull request merged**